content/docs/3_reference/3_panel/4_sections/0_pages/reference-panelsection.txt

```diff
@@ -123,6 +123,21 @@ layout: cards
 
 (image: cards.png)
 
+
+### Cardlets layout
+

```

Comment:
Same here, I would add it as snippet and reference it instead the same text in multiple pages